frontend/src/components/uam/uam.tsx

```diff
@@ -1,28 +1,14 @@
-import { FC, GroupsItemResponseDto } from 'common/types/types';
-import { useAppDispatch, useAppSelector, useEffect } from 'hooks/hooks';
-import { Column } from 'react-table';
-import { uamActions } from 'store/actions';
+import { FC } from 'common/types/types';
 
 import { GroupsTable, UsersTable } from './components/components';
-import { getGroupsColumns, getGroupsRows } from './helpers/helpers';
 import styles from './styles.module.scss';
 
 const UAM: FC = () => {
-  const dispatch = useAppDispatch();
-  const { groups } = useAppSelector((state) => state.uam);
-
-  useEffect(() => {
-    dispatch(uamActions.getGroups());
-  }, []);
-
-  const groupsColumns: Column<GroupsItemResponseDto>[] = getGroupsColumns();
-  const groupsRows: GroupsItemResponseDto[] = getGroupsRows(groups);
-
   return (
     <div className={styles.uam}>
       <h1 className={styles.pageTitle}>User Access Managment</h1>
       <UsersTable />
-      <GroupsTable data={groupsRows} columns={groupsColumns} />
+      <GroupsTable />
```

Comment:
Should not GroupsTable component receive it's data from higher level (UAM)?

Reply:
I think it would be better to have the data inside `GroupsTable`, because this data is not used in `UAM` for anything other than passing it down to `GroupsTable`